primer3/bindings.py

```diff
@@ -281,6 +300,11 @@ def calcTm(
         dv_conc: Divalent cation conc. (mM)
         dntp_conc: dNTP conc. (mM)
         dna_conc: DNA conc. (nM)
+        dmso_conc: Concentration of DMSO (%)
+        dmso_fact: DMSO correction factor, default 0.6
```

Comment:
Curious as to why the default for `dmso_fact` is included (while the defaults for other similar args are not?)

Reply:
Just copy pasta from the primer3 .c source